admin/src/app/home/orders/stock-orders/receive/services/indexdb.ts

```diff
@@ -0,0 +1,33 @@
+import Dexie from "dexie";
```

Comment:
There is another file that does the same thing 
`indexdb.service.ts`

Please delete this file and use the angular service instead

Reply:
Deleted